nbbuild/antsrc/org/netbeans/nbbuild/InsertModuleAllTargets.java

```diff
@@ -140,12 +144,12 @@ public void setUseClusters(boolean b) {
                         log("Cannot find build prerequisite " + cnb + " of " + entry, Project.MSG_WARN);
                         continue;
                     }
-                    String otherPath = other.getNetbeansOrgPath();
-                    if (otherPath == null) continue; // Do not add the all-module dependency for module which is in the binaries
-                    String otherCluster = clustersOfModules.get(otherPath);
+                    String otherId = other.getNetbeansOrgId();
+                    if (otherId == null) continue; // Do not add the all-module dependency for module which is in the binaries
+                    String otherCluster = clustersOfModules.get(otherId);
                     if (myCluster == null || otherCluster == null || myCluster.equals(otherCluster)) {
                         namedDeps.append(",all-");
-                        namedDeps.append(otherPath);
+                        namedDeps.append(otherId);
```

Comment:
The `all-xyz` tasks keep the names as they have them name - e.g. they have to use the module `id`.